### ISSUE TITLE
Problem installing python-pip for ansible?

### ISSUE THREAD

**Issue by bknowles** [daniellawrence/30second-devops#6]
I tried doing the 30-second demo for ansible, and got the following error:

```
$ docker build -t ansible-standalone .
Sending build context to Docker daemon 6.144 kB
Step 1 : FROM ubuntu:14.04
 ---> 6cc0fc2a5ee3
Step 2 : RUN apt-get install -y python-pip python-dev openssh-server
 ---> Running in bd18cec8c8c9
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package python-pip
The command '/bin/sh -c apt-get install -y python-pip python-dev openssh-server' returned a non-zero code: 100
```

Any ideas on how I can help with this one?

**Comment by bknowles** [comment]
I had an identical problem with installing fabric.  ;(

**Comment by daniellawrence** [comment]
This is going to be due to an outdated list/cache of packages for apt.
This commit should fix it https://github.com/daniellawrence/30second-devops/commit/2029dbaaf7493f130060a611c5e787bac09a3181

I'll test this later and then confirm if its fixed

**Comment by daniellawrence** [comment]
All fixed 865f83ac84ebd1cb7134417cd9072e29c8ae897d
